extensions/wikia/CategoryPage3/CategoryPage3Hooks.class.php

```diff
@@ -123,13 +123,51 @@ public static function onGetPreferences( $user, &$defaultPreferences ) {
 		return true;
 	}
 
-	static public function onUserGetDefaultOptions( &$defaultOptions ) {
+	public static function onLinkerMakeExternalLink( string $url, string $text, string &$link, array $attribs ): bool {
+		global $wgCityId, $wgContLang, $wgFandomBaseDomain, $wgWikiaBaseDomain;
+
+		$host = parse_url( $url, PHP_URL_HOST );
+		$host = wfNormalizeHost( $host );
+
+		// External-external links
+		if ( !endsWith( $host, $wgFandomBaseDomain ) && !endsWith( $host, $wgWikiaBaseDomain ) ) {
```

Comment:
This would pass for domains like `examplefandom.com`, right? It should check if it ends in `".{$wgFandomBaseDomain}" as we do elsewhere to be safe.